app/controllers/api/functions.php

```diff
@@ -228,6 +228,7 @@
             fn () => $dbForConsole->createDocument('schedules', new Document([
                 'region' => App::getEnv('_APP_REGION', 'default'), // Todo replace with projects region
                 'resourceType' => 'function',
+                'resourceCollection' => 'functions',
```

Comment:
So that we can use abstracted code to fetch resources for multiple types, didn't want to assume that the collection would always be `$resourceType . 's';`

Reply:
I would just handle a switch case according to the resource type, having both seems unnecessarily complex.

Reply:
But I do agree we shouldn't try and build the collection name.